README.md

```diff
@@ -85,19 +85,73 @@ The `href` can contain an optional server placeholder `{s}`. If `{s}` is used, t
 
 Links to a [PMTiles](https://github.com/protomaps/PMTiles/blob/main/spec/v3/spec.md) file (versions 3.x).
 
-| Field Name      | Type                 | Description |
-| --------------- | -------------------- | ----------- |
-| rel             | string               | **REQUIRED**. Must be set to `pmtiles`. |
-| href            | string               | **REQUIRED**. Link to a PMTiles file (usually ends with `.pmtiles`). |
-| type            | string               | Recommended to be set to `application/vnd.pmtiles`. |
-| pmtiles:layers  | \[string]            | For vector tiles, the layers to show on the map by default. If not provided, it's up to the discretion of the implementation to choose a layer from the `vector_layers` in the PMTiles metadata. |
+| Field Name               | Type                                           | Description |
+| ------------------------ | ---------------------------------------------- | ----------- |
+| rel                      | string                                         | **REQUIRED**. Must be set to `pmtiles`. |
+| href                     | string                                         | **REQUIRED**. Link to a PMTiles file (usually ends with `.pmtiles`). |
+| type                     | string                                         | Recommended to be set to `application/vnd.pmtiles`. |
+| pmtiles:layers           | \[string]                                      | For vector tiles, the layers to show on the map by default. If not provided, it's up to the discretion of the implementation to choose a layer from the `vector_layers` in the PMTiles metadata. |
+| pmtiles:layer_properties | Map\<string, Map\<string, JSON Schema Object>> | For vector tiles, the properties/fields available for each layer and their corresponding [JSON Schema](#json-schema-object) as an object. |
```

Comment:
Is this meant to be exhaustive of all layer properties existing in the tileset (like in TileJSON), or only the properties/fields we want to visualize (like in the `pmtiles:layers`)?

Reply:
Good question. I guess the second...

Reply:
OK, that seems much more feasible than the former, though TileJSON `vector_layers` does not provide information about the possible enum values etc so the JSON Schema objects need to be constructed with outside knowledge.

Reply:
Yes, indeed. If we would have enough information in vector_layers, we would not need this construct at all.